routes/index.js

```diff
@@ -29,11 +30,29 @@ routes.POSTregister = function(req, res, next) {
     if (err) {
       console.log('error while user register!', err); return next(err);
     }
-    console.log('user registered!');
-
-    passport.authenticate('local')(req, res, function () {
-      res.send({redirect: '/'});
+    
+    //send email verification
+    //Used example: https://github.com/heitortsergent/passport-local-mongoose-email/tree/master/examples/login
+    var authenticationURL = 'http://localhost:3000/verify?authToken=' + user.authToken;
+    sendgrid.send({
+      to:       user.email,
+      from:     'SpringInitiative@olinjs.com',
+      subject:  'Confirm your email for Spring Initiative',
+      html:     '<a target=_blank href=\"' + authenticationURL + '\">Confirm your email</a>'
+      }, function(err, json) {
+      if (err) { return console.error(err); }
     });
+    console.log("Email Sent");    
+  });
+}
+
+routes.GETemailver = function(req, res){
+  // User gets here after clicking the link in the email
+  // Hence needs to be redirected
+  User.verifyEmail(req.query.authToken, function(err, existingAuthToken) {
+    if(err) console.log('err:', err);
```

Comment:
If err should we also redirect to login page? What do they see if there's an error?